### ISSUE TITLE
Allow use of custom config.py

### ISSUE THREAD

**Issue by gutyex** [mide/minecraft-overviewer#39]
E.G Update entrypoint.sh to include:

```
FILE=/home/minecraft/config/config.py
if [ -f "$FILE" ]; then
    CONFIG = /home/minecraft/config/config.py
else 
    CONFIG = /home/minecraft/config.py
fi
```

And change the overviewer commands to
`--config $CONFIG`

I think this should allow custom config.py files by mounting a volume at /home/minecraft/config/ whist still defaulting to the included one if there's not one there.
Possibly there's a better way to do it that I'm missing.

**Comment by mide** [comment]
Very good idea. I will think about the various ways this could be implemented and open a PR.

**Comment by mide** [comment]
I have some ideas implemented but I need to spend a bit more time testing on this.

**Comment by mide** [comment]
Will be resolved via #41.
